dts/bindings/wifi/nordic,nrf7002-qspi.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: This is a representation of the nRF7002 WiFi chip.
+
+compatible: "nordic,nrf7002-qspi"
+
+include: "nordic,nrf7002.yaml"
+
+on-bus: qspi
+
+properties:
+  sck-frequency:
+    type: int
+    required: true
+    description: |
+      Maximum clock speed supported by the device, in Hz.
+
+  reg:
+    required: true
+
+  readoc:
+    type: string
+    required: false
+    enum:
+      - "fastread"
+      - "read2o"
+      - "read2io"
+      - "read4o"
+      - "read4io"
+    description: |
+      Specify the number of data lines and opcode used for reading.
+      If not specified `fastread` will be selected.
+      See below for meaning of each enumeration:
+        fastread - Single data line SPI, FAST_READ (0x0B)
+        read2o - Dual data line SPI, READ2O (0x3B)
+        read2io - Dual data line SPI, READ2IO (0xBB)
+        read4o - Quad data line SPI, READ4O (0x6B)
+        read4io - Quad data line SPI, READ4IO (0xEB)
+  writeoc:
+    type: string
+    required: false
+    enum:
+      - "pp"
+      - "pp2o"
+      - "pp4o"
+      - "pp4io"
+    description: |
+      Specify the number of data lines and opcode used for writing.
+      If not specified `pp` will be selected.
+      See below for meaning of each enumeration:
+        pp - Single data line SPI, PP (0x02)
+        pp2o - Dual data line SPI, PP2O (0xA2)
+        pp4o - Quad data line SPI, PP4O (0x32)
+        pp4io - Quad data line SPI, PP4IO (0x38)
+
+  address-size-32:
+    type: boolean
+    required: false
+    description: |
+      Set to indicate that 32-bit addressing is to be used.
+      If not specified 24-bit addressing will be used.
+
+  sck-delay:
+    type: int
+    required: false
+    default: 0
+    description: |
+      Number of clock cycles CSn must be asserted before it can go low
```

Comment:
This is a bit strange. Dunno how to fix, but the part "CSn must be asserted" does not sound right.
And what is the "it" that can go low again?

Reply:
`it` refers to CSn , asserted means high, so, we are saying we need to wait for sck-delay period before it can go low.

Reply:
Resolving based on @peknis  vote.